src/shared-components/banner/index.tsx

```diff
@@ -34,7 +34,11 @@ type BannerProps = {
  *
  * Banners are not dismissable.
  */
-export const Banner = ({ content, type = 'default', onClick }: BannerProps) => {
+export const Banner = ({
+  content,
+  type = 'default',
```

Comment:
[Nit] Let's alphabetize args to match newly alphabetized `BannerProps`.

Reply:
done